src/simulation.py

```diff
@@ -159,7 +160,7 @@ def process_advection(self, delta_time: float) -> None:
         delta_r /= const.POINT_SIDE_SIZE
 
         # buffering how far oil went in time step
-        self.advection_buffer += delta_r
+        self.advection_buffer += [delta_r[1], -delta_r[0]]
```

Comment:
double check

Reply:
delta_r - calculated from wind and current speed
speed is tuple - speed to north, speed to east <- so the first coord is Y not X as before